alyssabull/idk
# Feature/initial setup

## What does this PR do?
- Creates `App.js`, `RandomActivity.js` and `SavedActivities.js`
- Implements `Router` to bring users to three different views
- Fetches and displays data from the BoredAPI
- Displays a nav bar and buttons to navigate from page to page

## Where should the reviewer start?
- `App.js`, `RandomActivity.js` and `SavedActivities.js`

## How should this manually be tested?
- Generate a random activity by clicking the 'press for fun' button
- Generate a random activity on the random activity page
- Click the 'Home' button to be navigated back to the home page
- See a button conditionally render if a certain activity has an associated link
